src/sentry/migrations/0257_add_target_id_and_type_to_integrationfeature.py

```diff
@@ -0,0 +1,54 @@
+# Generated by Django 2.2.24 on 2021-12-09 00:52
+
+import django.db.models.deletion
+from django.db import migrations
+
+import sentry.db.models.fields.bounded
+import sentry.db.models.fields.foreignkey
+
+
+class Migration(migrations.Migration):
+    # This flag is used to mark that a migration shouldn't be automatically run in
+    # production. We set this to True for operations that we think are risky and want
+    # someone from ops to run manually and monitor.
+    # General advice is that if in doubt, mark your migration as `is_dangerous`.
+    # Some things you should always mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that
+    #   they can be monitored. Since data migrations will now hold a transaction open
+    #   this is even more important.
+    # - Adding columns to highly active tables, even ones that are NULL.
+    is_dangerous = False
+
+    # This flag is used to decide whether to run this migration in a transaction or not.
+    # By default we prefer to run in a transaction, but for migrations where you want
+    # to `CREATE INDEX CONCURRENTLY` this needs to be set to False. Typically you'll
+    # want to create an index concurrently when adding one to an existing table.
+    # You'll also usually want to set this to `False` if you're writing a data
+    # migration, since we don't want the entire migration to run in one long-running
+    # transaction.
+    atomic = True
+
+    dependencies = [
+        ("sentry", "0256_create_docintegration_table"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="integrationfeature",
+            name="target_id",
+            field=sentry.db.models.fields.bounded.BoundedBigIntegerField(null=True),
+        ),
+        migrations.AddField(
+            model_name="integrationfeature",
+            name="target_type",
+            field=sentry.db.models.fields.bounded.BoundedPositiveIntegerField(default=0),
```

Comment:
This table is tiny right? If so this should be safe.

Reply:
It's 20,689 rows, maybe not *tiny* but ... small?

Reply:
Yeah, that's small enough.